atom/browser/browser.h

```diff
@@ -293,6 +296,9 @@ class Browser : public WindowListObserver {
   // The browser is being shutdown.
   bool is_shutdown_ = false;
 
+  // TODO
```

Comment:
TODO?

Reply:
I wasn't sure what to put there, `Whether secure mode is enabled.` is not that useful